CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# What we are looking for
+
+Thank you for your interest in contributing! We are looking for additional podcasts to add to our expanding list.
+
+The current categories are:
+- Javascript
+- General Technology & Web Development 
+- Other Technology-Specific 
+- Women in Tech
+
+# To Contribute
+
+1. Fork the repository
```

Comment:
Maybe add a step to ask the contributors kindly, to check the list first so they would not add duplicates to the list